pages/rules/rules.css

```diff
@@ -0,0 +1,49 @@
+@import url("../../main.css");
+
+.rules-list {
+  width: 50%;
+  max-width: max-content;
+  background-color: var(--white-pure);
+  margin-top: 5rem;
+}
+
+.rules-head {
+  text-align: center;
+  font-size: 3.5rem;
```

Comment:
I see you are using `font-size:3.5rem;` in lots of places so consider making a utility class or something. That will be very useful in many scenarios. And same for `text-align:center;`.

Reply:
Sure, I will do it.